Add Community Guides footer with Render (#3931)
phoenixframework/phoenix: guides/deployment/deployment.md

  * [with Elixir's releases](releases.html)
  * [to Gigalixir](gigalixir.html), an Elixir-centric Platform as a Service (PaaS)
  * and [to Heroku](heroku.html), one of the most popular PaaS.

> ## Community Deployment Guides

Reviewer: ```suggestion
## Community Deployment Guides

```

  * [with Elixir's releases](releases.html)
  * [to Gigalixir](gigalixir.html), an Elixir-centric Platform as a Service (PaaS)
  * and [to Heroku](heroku.html), one of the most popular PaaS.

## Community Deployment Guides

> * [Render](https://render.com) has first class support for Phoenix applications, with no connection limits, no forced restarts and full support for Elixir clusters. Check out their guides for hosting Phoenix with [Mix releases](https://render.com/docs/deploy-phoenix), [Distillery](https://render.com/docs/deploy-phoenix-distillery), and as a [Distributed Elixir Cluster](https://render.com/docs/deploy-elixir-cluster).

Reviewer: ```suggestion
  * [Render](https://render.com) has first class support for Phoenix applications. There are guides for hosting Phoenix with [Mix releases](https://render.com/docs/deploy-phoenix), [Distillery](https://render.com/docs/deploy-phoenix-distillery), and as a [Distributed Elixir Cluster](https://render.com/docs/deploy-elixir-cluster).
```
